app/controllers/application_controller.rb

```diff
@@ -9,7 +9,10 @@ def current_user
 
 
   def current_admin?
-    current_user && current_user.admin?
+    platform_admin_role = Role.find_by(name: "Platform Admin")
```

Comment:
This method has been changed out to accommodate the old roles set up and incorporate the new roles set up. Ideally, we'll want to split out this and any further logic into some kind of Permission PORO.

Reply:
The User table actually has methods that check the user's role, (i.e. #platform_admin?, #store_admin?, etc.) perhaps we could utilized those methods here.

Reply:
Great call @izaxveta !